### ISSUE TITLE
Internal error on using scope defined/required by an interface

### ISSUE THREAD

**Issue by lptn** [nunomaduro/larastan#1663]
- Larastan Version: v2.6.3
- `--level` used: max

It happened after (v2.6.2 => v2.6.3) update

### Description

```
Internal error: Internal error: Method newEloquentBuilder() was not found in reflection of class App\Models\CanUseVideoInContent. in file  
     /app/Services/VideoLookup.php                                                                               
     Run PHPStan with -v option and post the stack trace to:                                                                                                  
     https://github.com/phpstan/phpstan/issues/new?template=Bug_report.yaml                                                                                   
     Child process error (exit code 1):                                                                                                                    
```

This happened when scope required by an interface

### Laravel code where the issue was found

see https://phpstan.org/r/33ec248e-1c68-42eb-baa7-1ffdf36ce2d8

I assume it's caused by https://github.com/nunomaduro/larastan/commit/eb31ffa9b5a755bd9ca2973aa93027e71c855d05

**Comment by canvural** [comment]
I think this is fixed by #1662

**Comment by lptn** [comment]
>I think this is fixed by https://github.com/nunomaduro/larastan/pull/1662

Not exactly: there is a new Internal error:
```
     Internal error: Internal error: Method newEloquentBuilder() was not found in reflection of class App\Modules\Video\Models\CanUseVideoInContent. in file  
     /Users/alies/code/app/Modules/Video/Services/VideoLookup.php                                                                               
     Run PHPStan with -v option and post the stack trace to:                                                                                                  
     https://github.com/phpstan/phpstan/issues/new?template=Bug_report.yaml                                                                                   
     Child process error (exit code 1): 
```
